### PR TITLE
Common: fixed non-option passing on custom() method

### PR DESCRIPTION
**Pull request by holgerd77**
Fixes a bug discovered in #1838

(open for everyone in the team to review of course)

### CODE REVIEW

File: packages/common/tests/customChains.spec.ts [hunk 1]
```diff
@@ -90,6 +90,20 @@ tape('[Common]: Custom chains', function (t: tape.Test) {
       )
     }
 
+    common = Common.custom(CustomChain.PolygonMumbai, { hardfork: Hardfork.Byzantium })
```
Comment (jochem-brouwer) [inline packages/common/tests/customChains.spec.ts:93]: Super nit but can you add a sanity check? I.e. (For the other one too). This ensures the integrity of these tests.

```
```suggestion
    common = Common.custom(CustomChain.PolygonMumbai)
    st.notEqual(common.hardfork(), Hardfork.Byzantium, 'by default PolygonMumbai hardfork is not Byzantium')
```

**Review by jochem-brouwer — COMMENTED** [review]
One comment, great that you took into this quickly closed issue 😄

**Comment by holgerd77** [comment]
Ok, have taken over the suggestion, yeah, thanks, makes a lot of sense to add! 🙂 

Might need another approval once tests pass.

**Review by ryanio — APPROVED** [review]
lgtm!

**Review by jochem-brouwer — APPROVED** [review]
Looks great, also great tests :smile:

**Pull request merged**